File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    answer = []
    for i in range(len(food)):
        for j in range(0,food[i]//2):
            answer.append(str(i))
    answer.append("0")
    answer+=sorted(answer[:-1],reverse=True)
    return ''.join(answer)

File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.kt
class Solution {
    fun solution(food: IntArray): String {
        var answer: String = ""
        for(i in 1 until food.size){
            repeat(food[i]/2){answer+=i}
        }
        var str = answer.reversed()
        answer+=0
        answer+=str
        
        return answer
    }
}